scipy/spatial/transform/rotation.py

```diff
@@ -1463,6 +1464,73 @@ def magnitude(self):
         else:
             return angles
 
+    def reduce(self, left=None, right=None, return_best=False):
+        """Reduce this rotation with the provided rotation groups.
+
+        Parameters
+        ----------
+        left : `Rotation` instance
+            Object containing the left rotation group.
+        right : `Rotation` instance
+            Object containing the right rotation group.
+        right : bool
+            Whether to return the best indices
+
+        Returns
+        -------
+        reduced : `Rotation` instance
+            Object containing reduced rotations.
+        best_indices: integer array
+            Best indices of `left` and `right`
```

Comment:
Explain that these are selected indices of rotations used for "reduction".

Reply:
I have replaced it with:
```
Indices of elements from `left` and `right` used for reduction.
```